packages/host/src/Footer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Box,
+  Home,
+  styled,
+  Typography,
+  useHostContext,
+  UserIcon,
+} from '@openmsupply-client/common';
+
+export const Footer: React.FC = () => {
+  const { user, store } = useHostContext();
+  const PaddedCell = styled(Box)({ display: 'flex', padding: '0 8px' });
```

Comment:
Can this go outside of the component? It's a bit weird here and is also going to remount rather than re-render  each time, unnecessarily